src/pipeline.js

```diff
@@ -324,10 +324,10 @@ class Pipeline {
      */
     const execTaps = async (taps, fnIdent, fnIdx) => {
       for (const [idx, t] of iter(taps)) {
-        const ident = getident(t, 'tap', idx);
-        logger.silly(`exec ${ident} before ${fnIdent}`);
+        const ident = `#${String(fnIdx).padStart(2, '0')}/${fnIdent}/tap-#${idx}`;
```

Comment:
You may take that opportunity to fix the type: `ident` => `indent`

Reply:
`ident` as in `identifier` is correct.

Reply:
Ah sorry, misread the code. I thought this was computing indentation 🤦‍♂